src/main/webapp/WEB-INF/tags/instructor/results/noResponsePanel.tag

```diff
@@ -27,7 +27,12 @@
                 <tbody>
                     <c:forEach items="${noResponsePanel.emails}" var="email">
                         <tr>
-                            <td>${fn:escapeXml(noResponsePanel.teams[email])}</td>
+                            <c:if test="${fn:escapeXml(noResponsePanel.teams[email])  == 'Instructors'}">
```

Comment:
A simple string comparison is not good in this case since this is coupled to the value of `Const.USER_TEAM_FOR_INSTRUCTOR`, and if that constant is changed, then this check will no longer work. Perhaps we can do this check by just checking if the email is an instructor's email?